presto-hive/src/test/resources/io/prestosql/plugin/hive/s3/security-mapping.json

```diff
@@ -46,6 +46,12 @@
       "group": "hq",
       "iamRole": "danny_hq_role"
     },
+    {
+      "prefix": "s3://endpointbucket",
```

Comment:
This also suffers the same comment: https://github.com/prestosql/presto/pull/3766#discussion_r427119728

I think we need a better way to injecting rules to tests. It would be nice to define them next to the test as java object, then do serialization round trip, and the do the test. Maybe this should go as a refactor before this commit or after.

Reply:
added, thanks!

Reply:
I agree, can look at refactoring after this commit.